nsq-mixin/lib/nsqgrafonnet/timeseries.libsonnet

```diff
@@ -0,0 +1,64 @@
+local grafana = import 'github.com/grafana/grafonnet-lib/grafonnet/grafana.libsonnet';
+local prometheus = grafana.prometheus;
+
+{
+  new(title, description=null)::
```

Comment:
This is great, I'll likely use it in some of my future mixins.

Do we anticipate building more of these for other newer visualizations?

Do we think that it makes sense to offer this upstream to grafonnet, or as a separate extension library?

Reply:
Even though there concerns about grafonnet lib(https://github.com/grafana/grafonnet-lib/issues/332), I think we should use some time to update grafonnet upstream.